stdlib/public/core/UTF8Span.swift

```diff
@@ -1,7 +1,8 @@
 // TODO: comment header
 
 
-/// TODO: docs
+/// `UTF8Span` represents a borrowed view into contiguous memory that
+/// contains validly-encoded UTF-8 code units.
```

Comment:
Omit type name in its abstract.

```suggestion
/// A borrowed view into contiguous memory that contains validly-encoded UTF-8 code units.
```